app/scrapers/stackoverflow.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import print_function
+from generalized import Scraper
+
+
+class StackOverflow(Scraper):
+    """Scrapper class for StackOverflow"""
+
+    def __init__(self):
+        self.url = 'https://stackoverflow.com/search'
+        self.result_url = 'https://stackoverflow.com'
```

Comment:
Justify `result_url` or remove it as it haven't been used in the exiisting scrapers in this project.

For page indexing use `self.startKey = 'page'`
Check https://stackoverflow.com/search?page=2&q=google

Reply:
I was using that as variable for appending the result link to result URL. Will remove it though. 
And @bhaveshAn if we go to the url you mentioned via browser, we can see the results. But when a scraper does that, its redirected to captcha page.
Please see the second image I just uploaded.

Reply:
Are you sure you are having idea about `self.startKey` and `self.defaultStart`

Reply:
yes, for stackoverflow the `startKey` is `page` and `defaultStart` is 1